corehq/apps/users/static/users/js/roles.js

```diff
@@ -72,6 +72,159 @@ hqDefine('users/js/roles',[
                 if (self.hasUnpermittedLocationRestriction) {
                     self.permissions.access_all_locations(true);
                 }
+                self.accessAreas = [
+                {
+                    editPermission: self.permissions.edit_web_users,
+                    viewPermission: self.permissions.view_web_users,
+                    text:  "<strong>Web Users</strong> &mdash; invite new web users, manage account settings, remove membership",
+                    showEditCheckbox: true,
+                    editCheckboxLabel: "edit-web-users-checkbox",
+                    showViewCheckbox: true,
+                    viewCheckboxLabel: "view-web-users-checkbox",
+                    srEditAndViewText: gettext("Edit & View Mobile Workers"),
```

Comment:
`sr` isn't a common prefix in our js. What would you think of renaming this to either `screenReaderEditAndViewText` or just `editAndViewText`?